doc/language_guide.md

```diff
@@ -260,6 +260,8 @@ TO PREDICT result_table_reference
 USING model_table_reference;
 ```
 
+**NOTE: The select statement must have the same logic used in training, e.g. you use `SELECT a, b * 2 FROM train_data TO TRAIN` to train a model, when predicting, you must use also `SELECT a, b * 2 FROM pred_data TO PREDICT` so that the model will behave properly. You should do the same when using `TO EXPLAIN` and `TO EVALUATE` clause.**
```

Comment:
- We don't need **NOTE** here.

----

Please be aware that the `SELECT ...` statement before the `TO PREDICT ..` clause must generate the same data schema as the one before the `TO TRAIN` clause does.  This requirement ensures that the data used in prediction has the same schema as those used in training.